doc/link-expansion.md

```diff
@@ -279,16 +277,16 @@ By default links contain the following fields:
 - `analytics_identifier` - Used to track a content item in analytics software
 - `api_path` - The path to the JSON representation of this item
 - `base_path` - The public path to this item
-- `content_id` - A UUID to represents the content item
-- `description` - A short description of the content item
-- `document_type` - The document of the content item
-- `locale` - The language the content item linked to is in
-- `public_updated_at` - The date/time that a content item was last changed
-- `schema_name` - The [GOV.UK content schema][govuk-content-schema] that a
-  content item conforms to
-- `title` - The title of a content item
+- `content_id` - A UUID to represents the document
+- `description` - A short description of the content
+- `document_type` - A document type - often identical to schema_name
```

Comment:
@MatMoore Sure - A link sounds a good idea. We've struggled explaining document_type in the past as it's quite a loose concept. Would be good to get it defined well in the docs.